commit 4a1a90ee1b2b4d13302046f043adf1c2a1de758d
Author: Raphael Kubo da Costa <rakuco@FreeBSD.org>
Date:   Thu Nov 26 15:36:26 2015 +0100

    Explicitly include CheckIncludeFiles.
    
    This fixes the build with CMake 3.4.0. We were calling check_include_files()
    and implicitly relying on CheckIncludeFiles being included.

--- CMakeLists.txt
+++ CMakeLists.txt
@@ -21,6 +21,7 @@ if(STRICT_BUILD)
                   -DQT_NO_URL_CAST_FROM_STRING -DQT_STRICT_ITERATORS)
 endif(STRICT_BUILD)
 
+include(CheckIncludeFiles)
 check_include_files(${CMAKE_CURRENT_SOURCE_DIR}/include/frontend.h HAVE_DVB)
 
 if(NOT HAVE_DVB)
